### PR TITLE
Quickjs build

### LINKED ISSUE #667: Build and link QuickJS runtime in CCF
https://bellard.org/quickjs/

### PR DESCRIPTION
**Pull request by achamayou**
fix #667 

This is just sorting out the build & linking.

### CODE REVIEW

File: src/apps/jsgeneric/jsgeneric.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+        // try find script for method
+        auto handler_script = scripts->get(args.method);
+        if (!handler_script)
+          return jsonrpc::error(
+            jsonrpc::StandardErrorCodes::METHOD_NOT_FOUND,
+            "No handler script found for method '" + args.method + "'");
+
+        const nlohmann::json response = {};
+        /*
```
Comment (olgavrou) [inline src/apps/jsgeneric/jsgeneric.cpp]: remove commented out code

**Comment by cimetrics** [comment]
quickjs-build@[3686 aka 20200107.14](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3686&view=results) vs master ewma over 20 builds from [3602](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3602&view=results) to [3676](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3676&view=results)
images

File: src/apps/jsgeneric/jsgeneric.cpp (hunk 1 above)
Comment (achamayou) [reply src/apps/jsgeneric/jsgeneric.cpp]: Well, this is sort of on purpose, it's the bit where the execution needs to go next :)

**Comment by cimetrics** [comment]
quickjs-build@[3692 aka 20200107.17](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3692&view=results) vs master ewma over 20 builds from [3603](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3603&view=results) to [3690](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3690&view=results)
images

**Pull request merged**